modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2701,6 +2702,56 @@ void ONNXImporter::parseGather(LayerParams& layerParams, const opencv_onnx::Node
     addLayer(layerParams, node_proto);
 }
 
+void ONNXImporter::parseGatherElements(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
+{
+    CV_CheckEQ(node_proto.input_size(), 2, "GatherElements: two inputs are required");
+
```

Comment:
Add a check here to check whether `indices` are provided as constant since the implementation relies on that:

```cpp
CV_CheckTrue(constBlobs.find(node_proto.input(1)) != constBlobs.end(), "DNN/ONNX-GatherElements: indices must be constant";
```

Reply:
Done